### DISCUSSION TITLE
Option for namespace tag to namespace only "id" and "for", but no "name" attribute

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#11467] [Ideas] [answered]
Would that be possible? I'm in the situation where i want to use the same form in multiple places and i want these forms to have different input ids, but share the same `name` values.

**Comment by brandonkelly** [comment, marked as answer]
You can do this with the [`namespaceAttributes`](https://craftcms.com/docs/4.x/dev/filters.html#namespaceattributes) filter.

**Reply by piotrpog** [reply]
Thanks for reply. Would be cool if this functionality also had twig tag.

**Reply by brandonkelly** [reply]
You can apply filters like normal tags using the [apply](https://twig.symfony.com/doc/3.x/tags/apply.html) tag.

```twig
{% apply namespaceAttributes %}
  <div id="foo">My HTML</div>
{% endapply %}
```

**Reply by piotrpog** [reply]
Oh, right. Forgot about that. Thanks for help.
